org.verapdf.releng/pom.xml

```diff
@@ -104,6 +104,13 @@
 					<groupId>org.eclipse.xtend</groupId>
 					<artifactId>xtend-maven-plugin</artifactId>
 					<version>${xtext.version}</version>
+					<dependencies>
+						<dependency>
+							<groupId>org.eclipse.platform</groupId>
+							<artifactId>org.eclipse.equinox.common</artifactId>
+							<version>3.10.200</version>
```

Comment:
Can we make this a single maven property please so we only change the version in one place, thanks.